package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start:dev": "nodemon ./bin/www",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "NODE_ENV=test && sequelize db:migrate:undo:all && sequelize db:migrate && sequelize db:seed:all && ./node_modules/.bin/mocha --reporter spec ./server/tests/*.js --recursive"
```

Comment:
`npm` should append `./node_modules/.bin` to your `PATH`. You don't have to fully qualify `mocha` here.